grpc/errors_test.go

```diff
@@ -0,0 +1,70 @@
+package grpc
```

Comment:
add a test for SimpleErrorScrubber and NewEncryptedErrorScrubber

Reply:
SimpleErrorScrubber is the default test

I will add one for the encrypted

Reply:
done